cypress/e2e/capidle/navigation.cy.js

```diff
@@ -28,7 +28,10 @@ describe("capidle navigation", () => {
     cy.get("main .guess").should("have.length", 1);
 
     // the only hint row is correctly populated
-    cy.get("main .guess > span:first-child()").should("have.text", "Adelaide");
+    cy.get("main .guess > span:first-child()").should(
```

Comment:
nit pick: consider making this statement one-line

Reply:
fixed (: